### PR TITLE
[DOC] adds solution to "no matches found" to troubleshoot section of install guide

### PR DESCRIPTION
**Pull request by AurumnPegasus**
#### Reference Issues/PRs

Adds to the documentation according to #2770 

#### What does this implement/fix? Explain your changes.

Added the solution to the troubleshoot section of ```installation.rst```.

##### For all contributions
- [x] The PR title starts with either [ENH], [MNT], [DOC], or [BUG] indicating whether the PR topic is related to enhancement, maintenance, documentation, or bug.

### CODE REVIEW

File: docs/source/installation.rst [hunk 1]
```diff
@@ -71,6 +71,19 @@ minimum dependencies and using an estimator which interfaces a package that has
...
 install ``sktime`` with maximum dependencies (see above).
 
+No matches found
+""""""""""""""""
+
+A frequent issue encountered when downloading the dev/all_extras version of dependencies
+using ``pip`` is *no matches found* error. This is mostly seen in Ubuntu/Linux systems, and is
+because the shell does not recognising most special characters as valid input.
+
+- Solution: use inverted commas:
```
Comment (fkiraly) [inline docs/source/installation.rst:81]: called quotation mark

**Review by fkiraly — CHANGES_REQUESTED** [review]
Thanks!

Although this might not be the right location, as the section above does not contain an instance of `all_extras,dev` as args?

**Comment by AurumnPegasus** [comment]
> Thanks!
> 
> Although this might not be the right location, as the section above does not contain an instance of `all_extras,dev` as args?

I will put it next to the place the command is given itself?

**Comment by fkiraly** [comment]
yes, that sounds like the best idea, or in an adjacent section.

File: docs/source/installation.rst (hunk 1 above)
Comment (AurumnPegasus) [reply docs/source/installation.rst]: Updated.

**Comment by AurumnPegasus** [comment]
@fkiraly 
Made changes as requested, please check and let me know if anything else is required

File: docs/source/installation.rst [hunk 2]
```diff
@@ -224,6 +224,16 @@ In the ``anaconda prompt`` terminal:
 
 Some users have experienced issues when installing NumPy, particularly version 1.19.4.
 
+A frequent issue encountered when downloading the dev/all_extras version of dependencies
```
[the remark this answers is not in the document]
Comment (fkiraly) [reply docs/source/installation.rst:227]: (it's also not a frequent issue)

[the remark this answers is not in the document]
Comment (fkiraly) [reply docs/source/installation.rst]: Thanks!

**Review by fkiraly — APPROVED** [review]
Looks good now, thanks!

**Pull request merged**